### PR TITLE
GEN-1336: chore: add prices check to PR template

### PR DESCRIPTION
**Pull request by fedealconada**
## Motivation

Context [here](https://www.notion.so/Oracles-9a744df2e7ef4159b0cca67be591efaa?d=50f26427368149b5a23acc656bb702ae#1fa00b8cb86549dea97bd68e6f711ad4)

## Implementer Checklist

- [ ]  [FIRST TIME ONLY] Review the [Solcurity](https://github.com/Rari-Capital/solcurity) standard
- [ ]  [OPTIONAL] Create a reference implementation (in python or JS)
- [ ]  Create a short "Motivation" section for the PR
- [ ]  Write a spec for the feature and put it in the PR description – basic function names and expected state transitions is OK
- [ ]  Check all of the new revert paths with concrete tests
- [ ]  Check all of the new storage slot writes with concrete tests – do not make the passing test case the trivial case!
- [ ]  Go line-by-line through the new code and ensure it has all been covered by concrete tests – since we don’t have a coverage engine for foundry yet, you can imagine how one might check that each different code path is covered
- [ ]  Simplify the implementation and spend some time trying to minimize gas costs
- [ ]  Write fuzz tests for the feature – try everything to break the implementation (is this function monotonically in/decreasing, should it always be less than something else, etc)
- [ ]  Capture bugs discovered during the above steps in concrete tests (regression tests)
- [ ]  Add integration tests – fuzz or concrete tests that test how the new code affects the entire system, either locally or with a mainnet fork
- [ ]  [OPTIONAL] Integrate the feature into the deployment scripts
- [ ]  [OPTIONAL] Add new sanity checks to the deployment scripts
- [ ]  Get the PR reviewed by at least two people

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -34,4 +34,10 @@ check off all of the required items.
...
 - [ ]  [OPTIONAL] Add new sanity checks to the deployment scripts
-- [ ]  Get the PR reviewed by at least two people
\ No newline at end of file
+- [ ]  Get the PR reviewed by at least two people
+
+If your PR uses or interacts with prices of any kind
+- [ ] Make sure forge is running the test on the latest block
+- [ ] Compare the value returned with some other source (like Curve)
+- [ ] Check the `updatedAt` (if available) value of the contract that is in fact sourcing the price is not older than X amount of time
+- [ ] Ensure oracle liveliness from the oracle maintainer, either through a public status interface or through written confirmation from a developer.
```
Comment (jparklev) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Ensure oracle liveness from the oracle maintainer, either through a public status interface or through written confirmation from a developer.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -34,4 +34,10 @@ check off all of the required items.
...
 - [ ]  [OPTIONAL] Integrate the feature into the deployment scripts
 - [ ]  [OPTIONAL] Add new sanity checks to the deployment scripts
-- [ ]  Get the PR reviewed by at least two people
\ No newline at end of file
+- [ ]  Get the PR reviewed by at least two people
+
+If your PR uses or interacts with prices of any kind
+- [ ] Make sure forge is running the test on the latest block
+- [ ] Compare the value returned with some other source (like Curve)
+- [ ] Check the `updatedAt` (if available) value of the contract that is in fact sourcing the price is not older than X amount of time
```
Comment (jparklev) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Check the `updatedAt` (if available) value of the contract and confirm that the price we are sourcing is in fact not older than X amount of time
```

**Review by jparklev — APPROVED** [review]
LGTM. I wonder if we should use this "conditional" format for the existing items as well (eg "if smart contract changes were made", "if this is your first time reviewing SC changes", etc)?

**Comment by fedealconada** [comment]
> LGTM. I wonder if we should use this "conditional" format for the existing items as well (eg "if smart contract changes were made", "if this is your first time reviewing SC changes", etc)?

how about now?

**Comment by jparklev** [comment]
> > LGTM. I wonder if we should use this "conditional" format for the existing items as well (eg "if smart contract changes were made", "if this is your first time reviewing SC changes", etc)?
> 
> how about now?

Nice, yea think it's an improvement

**Review by abylaw — APPROVED** [review]

**Comment by jparklev** [comment]
ready to merge @fedealconada?

**Pull request merged**